chain/chain/src/test_utils.rs

```diff
@@ -109,19 +116,23 @@ pub fn setup_with_tx_validity_period(
     .unwrap();
 
     let signer = Arc::new(create_test_signer("test"));
-    (chain, runtime, signer)
+    (chain, epoch_manager, runtime, signer)
 }
 
 pub fn setup_with_validators(
     vs: ValidatorSchedule,
     epoch_length: u64,
     tx_validity_period: NumBlocks,
-) -> (Chain, Arc<KeyValueRuntime>, Vec<Arc<InMemoryValidatorSigner>>) {
+) -> (Chain, Arc<KeyValueEpochManager>, Arc<KeyValueRuntime>, Vec<Arc<InMemoryValidatorSigner>>) {
```

Comment:
Maybe MockEpochManager/SimpleEpochManager is a better name? It doesn't always need to be used together with KeyValueRuntime.